components/camel-kafka/src/main/java/org/apache/camel/component/kafka/KafkaFetchRecords.java

```diff
@@ -176,7 +178,7 @@ private void setupCreateConsumerException(ForegroundTask task, int max) {
         String time = TimeUtils.printDuration(task.elapsed(), true);
         String topic = getPrintableTopic();
         String msg = "Gave up creating org.apache.kafka.clients.consumer.KafkaConsumer "
-                     + threadId + " to " + topic + " after " + max + " attempts (elapsed: " + time + ").";
+                + threadId + " to " + topic + " after " + max + " attempts (elapsed: " + time + ").";
```

Comment:
The format check may be complaining because of this line. If you look at it, you can see that it's different than the previous version.